### ISSUE TITLE
2013 Mac Pro-No Workflow Extension and No Workspace Option

### ISSUE THREAD

**Issue by tangierc** [latenitefilms/BRAWToolbox#2]
Per the TestFlight instructions, there's no workflow extension showing up nor anything in the Workspace window option. Filter is present however.  Using FCP 10.6.5
![Screen Shot 2022-12-23 at 2 21 33 PM](https://user-images.githubusercontent.com/9577806/209410804-4a11a26f-35a3-4a82-b36f-baeebc5003de.jpg)
![Screen Shot 2022-12-23 at 2 21 49 PM](https://user-images.githubusercontent.com/9577806/209410807-6da4538c-7d22-4a10-b412-2efc0df6be95.png)

**Comment by latenitefilms** [comment]
Thanks for reporting! I assume you've tried restarting your computer?

Sometimes macOS can get a bit funky with extensions. The only thing I can really suggest is try deleting Final Cut Pro and re-downloading from the App Store. Give that a shot and let me know if that fixes things?

**Comment by tangierc** [comment]
Thanks. Before going that far of a route, I tried **deleting my preferences. This worked**. Nothing is in workspaces menu, but I did manage to import 14 6K BRAW clips which were imported as synced clips with audio (LTC audio in my case).  The clips are not in the event in the photo because I already moved them out to another event.

I found a few other things. I don't know if they're bugs, but they're definitely worth a mention in terms of how FCP is supposed to work with clips. I'll do another post with those.

![Screen Shot 2022-12-23 at 3 58 12 PM](https://user-images.githubusercontent.com/9577806/209415102-4a9480dd-2994-4b93-b31f-cb605dd15aff.jpg)
